pdns/dnsdist-protobuf.hh

```diff
@@ -83,6 +87,8 @@
   };
   std::unordered_map<std::string, MetaValue> d_metaTags;
 
+  // FIXME wondering if the cost of moving to a shared_ptr would be that bad
```

Comment:
For `DNSQuestion` and `DNSResponse` it would mean always allocating them as a `shared_ptr` since we cannot just turn them into `shared_ptr` just-in-time, so the cost is likely significant.